source/apiVolontaria/volunteer/tests/tests_view_ParticipationsId.py

```diff
@@ -570,3 +570,6 @@ def test_delete_participation_that_doesnt_exist(self):
         self.assertEqual(json.loads(response.content), content)
 
         self.assertEqual(response.status_code, status.HTTP_404_NOT_FOUND)
+
+    def test_cell_manager_view_user_permission(self):
+        self.assertEqual(self.user_cell_manager.has_perm('auth.view_user'), True)
```

Comment:
This test doesn't make sens in this file. It's not related to the participation call on API

Reply:
Yes the cell manager tests are there, I'll copy the test setup in the tests_view_Users file